pkgs/top-level/all-packages.nix

```diff
@@ -12510,6 +12510,7 @@ with pkgs;
   gcc9Stdenv = overrideCC gccStdenv buildPackages.gcc9;
   gcc10Stdenv = overrideCC gccStdenv buildPackages.gcc10;
   gcc11Stdenv = overrideCC gccStdenv buildPackages.gcc11;
+  gcc12Stdenv = overrideCC gccStdenv buildPackages.gcc11;
```

Comment:
Isn't is a typo (`gcc11` instead of `gcc12`)?

Reply:
Good catch! Updated to `gcc12`.